test/system/002-setup-admin_spec.js

```diff
@@ -8,47 +11,32 @@ describe('Default Admin User', function () {
     // inbox - a local transport we can use to capture email without an SMTP server
     const inbox = new LocalTransport()
 
-    before(async function () {
-        // Create the FF application with a suitable test configuration
-        forge = await FF_UTIL.setupApp({
-            telemetry: { enabled: false },
-            driver: {
-                type: 'stub'
-            },
-            db: {
-                type: 'sqlite',
-                storage: ':memory:'
-            },
-            email: {
-                enabled: true,
-                transport: inbox
-            },
-            create_admin: true
-        })
-    })
+    const defaultConfig = {
+        telemetry: { enabled: false },
+        driver: {
+            type: 'stub'
+        },
+        db: {
+            type: 'sqlite',
+            storage: ':memory:'
+        },
+        email: {
+            enabled: true,
+            transport: inbox
+        },
+        create_admin: true
+    }
+
+    async function createFlowForgeApp (config) {
+        return await FF_UTIL.setupApp({ ...defaultConfig, ...config })
+    }
 
     after(function () {
         return forge.close()
```

Comment:
```suggestion
        return await forge.close()
```

This is probably the problem, you need to wait for the close to finish before the next test

Reply:
Ahh, this will also need a change to line 34 as well (which I can't do as a suggestion)

It will need to be:

```
    after( async function () {
        return await forge.close()
}
```